chainer/testing/function.py

```diff
@@ -418,21 +433,27 @@ def _check_forward_output_arrays_equal(
                 enumerate(zip(expected_arrays, actual_arrays))):
             try:
                 array_module.assert_allclose(expected, actual, **opts)
-            except AssertionError:
-                indices.append(i)
+            except AssertionError as e:
+                indices.append((i, e))
```

Comment:
The variable `indices` is not a list of indices, but an association list of errors.

Reply:
Removed this change from this PR.